hamster_gtk/overview/dialogs/export_dialog.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+
+# This file is part of 'hamster-gtk'.
+#
+# 'hamster-gtk' is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# 'hamster-gtk' is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with 'hamster-gtk'.  If not, see <http://www.gnu.org/licenses/>.
+
+
+"""This Module provides the ``ExportDialog`` class to choose file to be exported."""
+
+from __future__ import absolute_import
+
+import os.path
+from enum import Enum
+from gettext import gettext as _
+
+from gi.repository import Gtk
+
+ExportType = Enum('ExportType', ('CSV', 'ICAL', 'XML'))
+
+
+class ExportDialog(Gtk.FileChooserDialog):
+    """Dialog used for exporting."""
+
+    def __init__(self, parent):
+        """
+        Initialize headerbar.
+
+        Args:
+            parent (Gtk.Window): Parent window for the dialog.
+        """
+        super(ExportDialog, self).__init__(_("Please Choose where to export to"), parent,
+            Gtk.FileChooserAction.SAVE, (Gtk.STOCK_CANCEL, Gtk.ResponseType.CANCEL,
+                                         Gtk.STOCK_SAVE, Gtk.ResponseType.OK))
+
+        self._export_type_chooser = Gtk.ComboBoxText()
```

Comment:
It would be better to wrap all the ``type_chooser`` related code into a private method like ``self._get_type_choose`` that returns the deals with creating label and dropdown and just returns the grid.